doc/config.md

```diff
@@ -1285,6 +1285,32 @@ fast retries when multiple hosts are available.
 
 Default: `round-robin`
 
+### target_session_attrs
+
+Note: This configuration only applies to backend servers that report `hot_standby`
+or `default_transaction_read_only` in startup parameters (i.e. Postgres 14+).
+
+Only allow connections with certain properties, similar to libpq's `target_session_attrs`
+connection parameter. For example: given a comma-separated list of hosts where only is
+writable, `target_session_attrs=primary` will disconnect from any hosts that are not writable.
```

Comment:
The docs should note that `prefer-standby` is not supported. To be clear, it seems fine that it's not supported, I don't think we can easily implement this with the current way PgBouncer connects to servers.